akka-docs/src/main/paradox/typed/distributed-pub-sub.md

```diff
@@ -26,22 +26,25 @@ Additionally, add the dependency as below.
   version=AkkaVersion
 }
 
-## The Topic Actor
+## The Topic Registry
+
+Distributed publish subscribe is achieved by representing each pub sub topic with an actor, @apidoc[akka.actor.typed.pubsub.Topic](akka.actor.typed.pubsub.Topic$).
 
-Distributed publish subscribe is achieved by representing each pub sub topic with an actor, @apidoc[akka.actor.typed.pubsub.Topic](akka.actor.typed.pubsub.Topic$). 
+The topic actor needs to run on each node where subscribers will live or that wants to publish messages to the topic. 
 
-The topic actor needs to run on each node where subscribers will live or that wants to publish messages to the topic.
- 
-The identity of the topic is a tuple of the type of messages that can be published and a string topic name but it is recommended
-to not define multiple topics with different types and the same topic name.
+Topics can be looked up in the @apidoc[PubSub](akka.actor.typed.pubsub.PubSub$) registry, this way the same topic will be represented by the same actor for a whole
+actor system. If the topic has not yet been started.
```

Comment:
Is that last sentence incomplete?

Reply:
Yea, halfway through a thought. Fixed now.